client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -10,9 +10,7 @@
 package io.pravega.client.stream;
 
 import com.google.common.annotations.Beta;
```

Comment:
Can we also remove those API calls tagged with "Beta"? For instance `generateStreamCuts` has been there for the last 2 major versions at least. The same for the Byte API related methods (I also remember some Beta annotation there). I think that we can remove these `Beta` annotations at this point as many of these calls have been tested for quite a while so far. Also, I still see a deprecated call in this clasee: `resetReadersToCheckpoint`. Can we get rid of this one also in this PR to help users to use the right methods from 0.7 onwards?

Reply:
We should do that in a different PR.